src/components/CustomerModal/index.js

```diff
@@ -249,6 +294,43 @@ const CustomerModal = ({
 												placeholder="08 36 65 65 65"
 												style={{gridColumn: '2 / 4'}}
 											/>
+											<FormElem
+												{...props}
+												label="Son occupation"
```

Comment:
```suggestion
												label="Son poste"
```

Je pense que c'est plus francais poste que occupation